frontend/src/pages/api/subscribe.ts

```diff
@@ -0,0 +1,73 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { SendySubscribeForm } from "src/types/sendy";
+
+export type Data = {
+  message?: string;
+  error?: string;
+};
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  if (req.method !== "POST") {
+    return res.status(405).json({ error: "Method not allowed" });
+  }
+
+  const {
+    name = "",
+    email = "",
+    LastName = "",
+    hp = "",
+  } = JSON.parse(req.body as string) as SendySubscribeForm;
+
+  if (name === "" || email === "") {
+    return res
+      .status(400)
+      .json({ error: "Invalid input. Please provide name and email." });
+  }
+
+  try {
+    const sendyApiUrl = process.env.SENDY_API_URL || "";
+    const sendyApiKey = process.env.SENDY_API_KEY || "";
+    const list = process.env.SENDY_LIST_ID || "";
```

Comment:
If these aren't set, should it error/return instead? Otherwise I guess it would error somewhere in the request process (ie. if the URL is null we'd get some 404 error presumably?).

Reply:
Good call out, this is what the user sees if they try to submit the form without those values set:
![image](https://github.com/HHS/simpler-grants-gov/assets/25465464/a320b0c9-aea0-46a0-876d-17cbe39a13a1)